src/client/src/Client.c

```diff
@@ -385,6 +386,7 @@ STATUS getKinesisVideoMetrics(CLIENT_HANDLE clientHandle, PClientMetrics pKinesi
             CHK_STATUS(contentViewGetAllocationSize(pKinesisVideoClient->streams[i]->pView, &viewAllocationSize));
             pKinesisVideoMetrics->totalContentViewsSize += viewAllocationSize;
             pKinesisVideoMetrics->totalFrameRate += (UINT64) pKinesisVideoClient->streams[i]->diagnostics.currentFrameRate;
+            pKinesisVideoMetrics->totalElementaryFrameRate += pKinesisVideoClient->streams[i]->diagnostics.elementaryFrameRate;

```

Comment:
Before adding this you need to check for the struct version.  Only V1 of the struct will have this field.

I noted below the tests should test correct behavior for the latest version as well as older versions of the struct which don't have this field to make sure it doesn't break since this is a public API.

Reply:
@hassanctech , 
As discussed, these are purely API tests. The testes with the tag `_Valid` test validity of all versions.